### PR TITLE
[GLUTEN-1367] Support running gluten on anolis

### PR DESCRIPTION
**Pull request by leesf**
## What changes were proposed in this pull request?

(Fixes: \#1367)

## How was this patch tested?

manual tests on anolis

### CODE REVIEW

**Comment by FelixYBW** [comment]
@Yohahaha

File: backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala [hunk 1]
```diff
@@ -40,6 +40,10 @@ class VeloxInitializerApi extends InitializerApi {
         new VeloxSharedlibraryLoaderCentos7
       } else if (system.contains("alinux") && system.contains("3")) {
         new VeloxSharedlibraryLoaderCentos8
+      } else if (system.contains("Anolis") && system.contains("8")) {
+        new VeloxSharedlibraryLoaderCentos8
+      } else if (system.contains("Anolis") && system.contains("7")) {
```
Comment (Yohahaha) [inline backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala:45]: Alinux3 is verified with Alibaba cloud EMR release version [5.11.1](https://developer.aliyun.com/article/1182969?spm=5176.21211227.J_6302206100.1.7a6c123ciSdv41)
I'm not sure Anolis7/8 is supported too.

How do you test it?

Comment (leesf) [reply backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala:45]: @Yohahaha Thanks for your review, we use our own basic image with anolis 8 with spark and only modify the VeloxInitializerApi in gluten and do not modify centos set.up script.

Comment (Yohahaha) [reply backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala:45]: Thanks for clarify, make sense for me.
cc @FelixYBW @zhejiangxiaomai

**Review by FelixYBW — APPROVED** [review]

**Comment by leesf** [comment]
@FelixYBW this PR would be landed, thanks.

**Review by zhejiangxiaomai — APPROVED** [review]

**Comment by zhejiangxiaomai** [comment]
If CI pass, I will merge this pr.

**Comment by zzcclp** [comment]
retest this please

**Comment by zhejiangxiaomai** [comment]
> retest this please

It seems that there is still a problem with retest.

**Comment by zzcclp** [comment]
test this please

**Pull request merged**